src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -89,6 +89,15 @@ export type SnapshotAccessRequest = {
   datasetRequest: SnapshotBuilderRequest;
 };
 
+export const convertDomainOptionToConceptSet = (
+  domainOption: SnapshotBuilderDomainOption
+): SnapshotBuilderDatasetConceptSet => {
+  return {
+    name: domainOption.name,
+    featureValueGroupName: domainOption.name,
+  };
+};
+
```

Comment:
Just to make sure I understand: This will change when we adjust snapshot builder settings (https://broadworkbench.atlassian.net/browse/DC-1120), right?

Reply:
We will need to adjust it, yeah.

Reply:
So yes, this works with the domains pulled from the "domainOptions" section of the settings. Which, looking at your changes, it looks like this section isn't going to change. But, this just converts the domain options to concept sets. And looking at your changes, it looks like the concept sets will change so the featureValueGroupName won't be needed anymore. Does that sound right?